pandas/tests/frame/test_combine_concat.py

```diff
@@ -207,7 +210,10 @@ def test_update(self):
         other = DataFrame([[3.6, 2., np.nan],
                            [np.nan, np.nan, 7]], index=[1, 3])
 
-        df.update(other)
+        if inplace:
+            df.update(other, inplace=inplace)
+        else:
+            df = df.update(other, inplace=inplace)
```

Comment:
this is not an actual test, you need to rename to something else and assert that the original is not changed

Reply:
that's a valid objection - though I haven't seen it anywhere on the tests I've worked on that had an inplace-kwarg.